javascript/packages/orchestrator/src/providers/k8s/substrateCliArgsHelper.ts

```diff
@@ -0,0 +1,87 @@
+import { series } from "@zombienet/utils";
+import { ComputedNetwork, SubstrateCliArgsVersion } from "../../types";
+import { getClient } from "../client";
+import { createTempNodeDef, genNodeDef } from "./dynResourceDefinition";
+import { KubeClient } from "./kubeClient";
+
+const debug = require("debug")("zombie::substrate-cli-helper");
+
+const getVersion = async (
+  image: string,
+  command: string,
+): Promise<SubstrateCliArgsVersion> => {
+  const client = getClient() as KubeClient;
+  const fullCmd = `${command} --help | grep ws-port`;
+  const node = await createTempNodeDef(
+    "temp",
+    image,
+    "", // don't used
+    fullCmd,
+    false,
+  );
+
+  const podDef = await genNodeDef(client.namespace, node);
+  const podName = podDef.metadata.name;
+  await client.spawnFromDef(podDef);
+  const logs = await client.getNodeLogs(podName);
+  debug("logs:");
+  debug(logs);
+
+  return logs.includes("--ws-port <PORT>")
+    ? SubstrateCliArgsVersion.V0
+    : SubstrateCliArgsVersion.V1;
+};
+
+export const setSubstrateCliArdsVersion = async (network: ComputedNetwork) => {
+  // Calculate substrate cli version for each node
+  // and set in the node to use later when we build the cmd.
+  const imgCmdMap = new Map();
+  network.relaychain.nodes.reduce((memo, node) => {
+    const uniq_image_cmd = `${node.image}_${node.command}`;
+    if (!memo.has(uniq_image_cmd))
+      memo.set(uniq_image_cmd, { image: node.image, command: node.command });
+    return memo;
+  }, imgCmdMap);
+
+  network.parachains.reduce((memo, parachain) => {
+    for (const collator of parachain.collators) {
+      const uniq_image_cmd = `${collator.image}_${collator.command}`;
+      if (!memo.has(uniq_image_cmd))
+        memo.set(uniq_image_cmd, {
+          image: collator.image,
+          command: collator.command,
+        });
+    }
+    return memo;
+  }, imgCmdMap);
+
+  // check versions in series
+  const promiseGenerators = [];
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  for (const [_k, v] of imgCmdMap) {
```

Comment:
You can skip values in destructuring:

```suggestion
  for (const [, v] of imgCmdMap) {
```